### ISSUE TITLE
inSpec intermittently unable to expand tilde

### ISSUE THREAD

**Issue by cliffano** [shinesolutions/aem-helloworld-custom-stack-provisioner#1]
Pre common step intermittently fails due to inspec (via pry) attempting to expand path that includes tilde in order to resolve pry config (which is not used at all in this scenario), but it fails to expand tilde.

```
[   87.266265] cloud-init[3719]: /opt/puppetlabs/puppet/lib/ruby/gems/2.4.0/gems/pry-0.12.2/lib/pry/pry_class.rb:5:in `expand_path': couldn't find login name -- expanding `~' (ArgumentError)
[   87.266648] cloud-init[3719]: from /opt/puppetlabs/puppet/lib/ruby/gems/2.4.0/gems/pry-0.12.2/lib/pry/pry_class.rb:5:in `<class:Pry>'
[   87.267154] cloud-init[3719]: from /opt/puppetlabs/puppet/lib/ruby/gems/2.4.0/gems/pry-0.12.2/lib/pry/pry_class.rb:1:in `<top (required)>'
[   87.267708] cloud-init[3719]: from /opt/puppetlabs/puppet/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
[   87.268297] cloud-init[3719]: from /opt/puppetlabs/puppet/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
[   87.268834] cloud-init[3719]: from /opt/puppetlabs/puppet/lib/ruby/gems/2.4.0/gems/pry-0.12.2/lib/pry.rb:119:in `<top (required)>'
[   87.269420] cloud-init[3719]: from /opt/puppetlabs/puppet/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
[   87.269970] cloud-init[3719]: from /opt/puppetlabs/puppet/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
[   87.270555] cloud-init[3719]: from /opt/puppetlabs/puppet/lib/ruby/gems/2.4.0/gems/inspec-1.51.6/lib/inspec/shell.rb:5:in `<top (required)>'
[   87.272519] cloud-init[3719]: from /opt/puppetlabs/puppet/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
[   87.273008] cloud-init[3719]: from /opt/puppetlabs/puppet/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
[   87.273566] cloud-init[3719]: from /opt/puppetlabs/puppet/lib/ruby/gems/2.4.0/gems/inspec-1.51.6/lib/inspec.rb:15:in `<top (required)>'
[   87.274207] cloud-init[3719]: from /opt/puppetlabs/puppet/lib/ruby/gems/2.4.0/gems/inspec-1.51.6/bin/inspec:10:in `require_relative'
[   87.274763] cloud-init[3719]: from /opt/puppetlabs/puppet/lib/ruby/gems/2.4.0/gems/inspec-1.51.6/bin/inspec:10:in `<top (required)>'
[   87.275174] cloud-init[3719]: from /opt/puppetlabs/puppet/bin/inspec:23:in `load'
[   87.275738] cloud-init[3719]: from /opt/puppetlabs/puppet/bin/inspec:23:in `<main>'
```

**Comment by cliffano** [comment]
Change commit: https://github.com/shinesolutions/aem-helloworld-custom-stack-provisioner/commit/344e6657dadbf8afbbffa41072df5b32cd6643e1

**Comment by cliffano** [comment]
Done. Included in 0.10.0 .
